### PR TITLE
Fix/717 location issue

### PR DESCRIPTION
**Pull request by sparks2209**
Fixed bug associated with iOS, when location is turn off in settings of phone unable to complete app permission.

In app for location we are handling Granted and Blocker Scenarios and Not managing UNKnown status.

Code changed for default permission status to Denied when Granted this property is assigned again.

Rest of the cases like Blocked, Denied and Unknown will have status as DENIED.

### CODE REVIEW

**Review by johnschoeman — COMMENTED** [review]
Thanks for grabbing this! sadly i think it will conflict with the work done here: https://github.com/Path-Check/covid-safe-paths/pull/909
So i think you'll need to wait for that to merge and rebase and/or reintroduce this fix.
Sorry for this, I think we could be doing a better job at not letting prs stay open for so long!

**Comment by sparks2209** [comment]
@johnschoeman : Thank you for letting me know this, Let me know once your code available in develop branch. I will take code from develop and resolve conflicts.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,25 @@
-## [1.0.11](https://github.com/Path-Check/covid-safe-paths/compare/v0.5.4...1.0.11) (2020-05-13)
+## [1.0.13](https://github.com/sparks2209/covid-safe-paths/compare/1.0.12...1.0.13) (2020-05-21)
```
Comment (tstirrat) [inline CHANGELOG.md:1]: Undo these changes to this file

Comment (sparks2209) [reply CHANGELOG.md:1]: Do mean to keep CHANGELOG.md file empty? or just to deleted above added line.

**Comment by sparks2209** [comment]
With latest changes, user is not blocked in permission screen, when device location is off in settings of device.

One of the wired thing on app launch after resetting simulator, notification permission by default showing as Denied.
As we don't have any icon for Unknown/Unavailable status location permission icon is empty.

Simulator Screen Shot - iPhone 11 - 2020-05-29 at 16 13 14

We can close this pull request as user is not blocked.

@summetj @johnschoeman FYI
